src/lib/game/world/handler.js

```diff
@@ -112,16 +114,7 @@ class WorldHandler extends EventEmitter {
     });
   }
 
-  changeModel(_unit, oldModel, newModel) {
-    // Only need to handle skeleton helper changes here
-    if (oldModel && oldModel.skeletonHelper) {
-      this.scene.remove(oldModel.skeletonHelper);
-    }
-
-    if (newModel) {
-      newModel.skeletonHelper = new THREE.SkeletonHelper(newModel);
-      this.scene.add(newModel.skeletonHelper);
-    }
+  changeModel(_unit, _oldModel, _newModel) {
```

Comment:
I'm assuming this is no longer needed?

Had been using the following to test out display IDs: `player.displayID = 12831`